checkmarx/project.py

```diff
@@ -285,3 +81,57 @@ def scan_finished():
 
     def scan_statistics(self, scan_id: int):
         return self.client.get_scan_statistics(scan_id=scan_id)
+
+    def get_last_scans(self):
+        return self.client.get_last_scans_of_project(project_id=self.project_details.id)
+
+    def is_scan_finished(self, scan: model.ScanResponse):
+        if model.ScanStatusValues(scan.status.id) in (
+                model.ScanStatusValues.FINISHED,
+                model.ScanStatusValues.FAILED,
+                model.ScanStatusValues.CANCELED,
+        ):
+            return True
+        else:
```

Comment:
rather explicitly list the enum-values for `False`-case, and `raise NotImplementedError` in `else`

Reply:
I've added this